backend/src/main/java/com/sim_backend/websockets/exceptions/IllegalAuthorizeResponse.java

```diff
@@ -0,0 +1,7 @@
+package com.sim_backend.websockets.exceptions;
+
+public class IllegalAuthorizeResponse extends RuntimeException {
```

Comment:
I don't think this is required. Issues with serializing / empty messages should be handled by the WebSocket client code. If I'm missing something and it is required, document the reasoning in the code like other exceptions.